modules/encrypt/tests/EncryptMcryptTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * @group      kohana
+ * @group      kohana.encrypt
+ * 
+ * @package    Kohana/Encrypt
+ * @category   Test
+ * @author     Koseven Team
+ * @copyright  (c) Kohana Team
+ * @license    https://koseven.ga/LICENSE.md
+ */
+class EncryptMcryptTest extends EncryptTestBase
+{
+	/**
+	 * @return void
+	 */
+    public function setUp()
+    {
+        parent::setUp();
+
+		if ( ! extension_loaded('mcrypt'))
+		{
+			$this->markTestSkipped('The Mcrypt extension is not available.');
+		}
+
+        $this->set_config([
+            'type' => Encrypt_Engine_Mcrypt::TYPE,
+            'key' => EncryptTestBase::KEY32,
+        ]);
+	}
+
+	/**
+	 * @dataProvider provider_encode_and_decode
+	 * @return void
+	 */
+	public function test_256_bit(string $encryptable)
+	{
+        $this->set_config([
+            'type' => Encrypt_Engine_Mcrypt::TYPE,
+            'cipher' => MCRYPT_RIJNDAEL_256,
```

Comment:
MCRYPT_RIJNDAEL_256 is not AES256
AES256 is MCRYPT_RIJNDAEL_128 with 32bit key

Reply:
Yes you are right! thats one of the reasons why mcrypt is so bad:
https://paragonie.com/blog/2015/05/if-you-re-typing-word-mcrypt-into-your-code-you-re-doing-it-wrong#title.1.2

I will fix it with my next commit thanks!

Reply:
Ok I fixed this